src/AccessibilityInsights.Extensions.AzureDevOps/Properties/AssemblyInfo.cs

```diff
@@ -27,7 +27,7 @@
 [assembly: NeutralResourcesLanguage("en-US")]
 
 #if ENABLE_SIGNING
-[assembly: InternalsVisibleTo("AccessibilityInsights.Core.Fakes,PublicKey=0024000004800000940000000602000000240000525341310004000001000100e92decb949446f688ab9f6973436c535bf50acd1fd580495aae3f875aa4e4f663ca77908c63b7f0996977cb98fcfdb35e05aa2c842002703cad835473caac5ef14107e3a7fae01120a96558785f48319f66daabc862872b2c53f5ac11fa335c0165e202b4c011334c7bc8f4c4e570cf255190f4e3e2cbc9137ca57cb687947bc")]
+[assembly: InternalsVisibleTo("AccessibilityInsights.Extensions.AzureDevOps.Fakes,PublicKey=0024000004800000940000000602000000240000525341310004000001000100e92decb949446f688ab9f6973436c535bf50acd1fd580495aae3f875aa4e4f663ca77908c63b7f0996977cb98fcfdb35e05aa2c842002703cad835473caac5ef14107e3a7fae01120a96558785f48319f66daabc862872b2c53f5ac11fa335c0165e202b4c011334c7bc8f4c4e570cf255190f4e3e2cbc9137ca57cb687947bc")]
```

Comment:
it means that we sign the Fakes file at least with strong name key, do we?

Reply:
Apparently, yes.